### PR TITLE
[FLOC-4255] Add documentation for Flocker deploy on RHEL 7.2

### PR DESCRIPTION
**Pull request by myechuri**
Add documentation for installing flocker node and flocker cli on RHEL 7.2. Relevant sections start at http://clusterhq-staging-docs.s3-website-us-east-1.amazonaws.com/add-rhel-documentation-FLOC-4255/docker-integration/manual-install.html

Setting up RHEL CI environment will follow this PR.

### CODE REVIEW

File: docs/installation/install-node.rst [hunk 1]
```diff
@@ -44,6 +44,54 @@ If you do not have any nodes, the following guides will help you set some up, wi
...
+      ssh root@<your-first-node>
+
+#. **Install the** ``clusterhq-flocker-node`` **package:**
+
+   To install ``clusterhq-flocker-node`` on RHEL 7 you must install the RPM package provided by the ClusterHQ repository.
+   The commands below will install the two repositories and the ``clusterhq-flocker-node`` package.
+   
+   Run the following commands as root on the target node:
+
+   .. prompt:: bash root@rhel:~$
```
Comment (davidoliverSP2) [inline docs/installation/install-node.rst]: If this is run as root, should the prompt directive be `.. prompt:: bash [root@rhel]#`?

File: docs/installation/install-node.rst [hunk 2]
```diff
@@ -44,6 +44,54 @@ If you do not have any nodes, the following guides will help you set some up, wi
...
+#. **Install the** ``clusterhq-flocker-docker-plugin`` **package:**
+
+   At this point you can choose to install the Flocker plugin for Docker.
+   Run the following command as root on the target node:
+
+   .. prompt:: bash [root@rhel]#
+   
+      yum install -y clusterhq-flocker-docker-plugin
+
+   .. XXX FLOC-3454 to create a task directive for installing the plugin
```
Comment (davidoliverSP2) [inline docs/installation/install-node.rst]: I added indents here (and below where the same comment was used), as it was causing the numbering to get misaligned.

File: docs/installation/enabling-control-service.rst [hunk 3]
```diff
@@ -29,6 +29,27 @@ For more details on configuring the firewall, see the `FirewallD documentation`_
 
 On AWS, an external firewall is used instead, which will need to be configured similarly.
 
+RHEL 7.2
+========
+
+.. prompt:: bash root@rhel:~$
```
Comment (davidoliverSP2) [inline docs/installation/enabling-control-service.rst]: The other instructions in this document use prompt directives using root - should this be `.. prompt:: bash [root@rhel]#?`

**Comment by davidoliverSP2** [comment]
Thanks @myechuri 

I added a few commits that cleaned up some indenting - I added an indent to a comment (which I don't think you added in this PR), and removed some indents from code blocks.

File: docs/installation/install-node.rst (hunk 1 above)
Comment (myechuri) [inline docs/installation/install-node.rst]: Thanks for pointing out! Fixed.

File: docs/installation/install-node.rst (hunk 2 above)
Comment (myechuri) [inline docs/installation/install-node.rst]: Thanks!

File: docs/installation/enabling-control-service.rst (hunk 3 above)
Comment (myechuri) [inline docs/installation/enabling-control-service.rst]: Fixed.

**Comment by myechuri** [comment]
Thanks for the quick review, @davidoliverSP2 ! Addressed your review comments.

File: docs/docker-integration/install-node.rst [hunk 4]
```diff
@@ -21,6 +21,12 @@ Installing the Flocker Node Services
 * :ref:`aws-install-docker`
 * :ref:`rackspace-install-docker`
 
+.. _rhel-7-install-docker:
```
Comment (davidoliverSP2) [inline docs/docker-integration/install-node.rst]: Does this need to be replicated in the Kubernetes and Standalone instructions? Currently the RHEL instructions for Installing the Node services are only appearing in the Docker integration instructions.

File: docs/installation/install-client.rst [hunk 5]
```diff
@@ -57,6 +57,19 @@ On Ubuntu 14.04 (64-bit), the Flocker CLI can be installed from the ClusterHQ re
 
 .. begin-body-installing-client-linux
 
+Installing on RHEL 7.2
```
Comment (davidoliverSP2) [inline docs/installation/install-client.rst]: These instructions have been embedded into the `installing-client-linux` section - I'm not sure if this was intentional - it probably makes sense unless there is a scenario where we would want to publish instructions for "other linux distributions" but not RHEL. 

As it stands, every version of our install instructions (Docker/Kubernetes/Standalone) all have RHEL in the installing the client, but not all of them have RHEL in the node instructions (see my previous comment)

**Comment by davidoliverSP2** [comment]
Thanks @myechuri 

I've raised a few comments relating to whether the RHEL instructions need to appear for both installing the client and node services - currently they appear in all of the client instructions, but only the Docker integration for the node services.

I would suggest best practice is to label up the RHEL instructions in the `installation/install-client.rst` file as you have in the `installation/install-node.rst` document, then in each of the integration folders add an include directive. 

If you'd rather pass this over to me, I'm happy to pick it up - just move the Jira ticket to me!

File: docs/docker-integration/install-node.rst (hunk 4 above)
Comment (myechuri) [inline docs/docker-integration/install-node.rst]: Thanks for pointing out, @davidoliverSP2 . Added.

**Comment by myechuri** [comment]
> I would suggest best practice is to label up the RHEL instructions in the installation/install-client.rst file as you have in the installation/install-node.rst document, then in each of the integration folders add an include directive.

Thanks for suggesting, @davidoliverSP2 . Refactored client install instructions accordingly - definitely makes it cleaner.

**Comment by myechuri** [comment]
@davidoliverSP2 : addressed your comments and verified that the staging results comply: http://clusterhq-staging-docs.s3-website-us-east-1.amazonaws.com/add-rhel-documentation-FLOC-4255

Please let me know if you have any further comments. Thanks!

**Pull request merged**